app/controllers/screenings_controller.rb

```diff
@@ -4,16 +4,13 @@ class ScreeningsController < ApplicationController
   include TmdbHandler
 
   before_action :authenticate_user!
-  before_action :set_movie
-  before_action :set_screening, only: [:show, :edit, :update, :destroy]
+  before_action :set_movie, only: [:index, :new, :edit, :create, :update, :destroy]
+  before_action :set_screening, only: [:edit, :update, :destroy]
 
   def index
     @screenings = @movie.screenings.by_user(current_user)
   end
 
-  def show
-  end
```

Comment:
We don't use the show page for screenings, so I removed:
* this action
* the route
* the view